### PR TITLE
Holopads have a special animation for an unanswered incoming call

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax, MrDoomBringer
sprite: Holopads now flash green when there is an incoming call.
/:cl:

Why? Sometimes you can be in "earshot" of more than one holopad, and it's useful to know which one is actually ringing.

### CODE REVIEW

**Comment by WJohn** [comment]
While I am in favor of finally having a visual tell for which holopad is ringing, this is not a superb animation at all. A blinking and largely static green stripe down the middle (which is really actually kind of off center looking because of how holopads are drawn) is not all that pretty. I was hoping for at least for the bar to sweep across the holopad, or maybe for the whole thing to flash green. Something like that.

**Comment by coiax** [comment]
@WJohn spriting-senpai, please make my codersprites prettier

**Comment by MrDoomBringer** [comment]
holopad1
holobig

Decided to give a stab at it, @WJohn hows this?

**Comment by vuonojenmustaturska** [comment]
I think the cyan is too saturated and i don't think it's a "nanotrasen-ish" color to begin with. Up to wjohn though.

**Comment by MrDoomBringer** [comment]
@vuonojenmustaturska eh maybe. I used the same color as coiax as wj didnt mention it

**Comment by ExcessiveUseOfCobblestone** [comment]
have a holograph of a phone ringing :^)

**Comment by Cyberboss** [comment]
@coiax

**Comment by coiax** [comment]
I mean, I'm waiting for our sprite master to approve @MrDoomBringer's sprite.

**Comment by WJohn** [comment]
Oh right this is a thing. It's better now, I think it'd be better still if it flashed twice in quick succession followed by a delay in its animation. A steady constant pulsing isn't what comes to mind for me personally for what amounts to phone ring.

**Comment by MrDoomBringer** [comment]
will do

**Comment by Cyberboss** [comment]
@MrDoomBringer

**Comment by MrDoomBringer** [comment]
oh shit whup

**Comment by MrDoomBringer** [comment]
holozig

**Comment by coiax** [comment]
Am aware of new sprite, will update this PR when I am not dying of the common cold.

File: code/game/machinery/hologram.dm [hunk 1]
```diff
@@ -339,7 +342,9 @@ Possible to do for anyone motivated enough:
 				HC.Disconnect(src)//can't answer calls while calling
 			else
 				playsound(src, 'sound/machines/twobeep.ogg', 100)	//bring, bring!
+				ringing = TRUE
 
+	SetLightsAndPower()
```
Comment (AnturK) [inline code/game/machinery/hologram.dm]: Is this going to try to update light source every process ?

Comment (coiax) [reply code/game/machinery/hologram.dm]: Currently, yes. I could just call update_icon().

Are repeated calls of set_light(X) expensive? I thought it wouldn't recalculate if it was the same value.

Comment (AnturK) [reply code/game/machinery/hologram.dm]: Not horribly since it will pull out on required update check most likely, but it's still few needless procalls

Comment (ExcessiveUseOfCobblestone) [reply code/game/machinery/hologram.dm]: You could check to see if ringing is the same as when the proc started before doing the update.

Comment (optimumtact) [reply code/game/machinery/hologram.dm]: PLease to do the needful

**Pull request merged**